pkg/collector/corechecks/snmp/config_validate_enrich.go

```diff
@@ -48,20 +48,28 @@ func validateEnrichMetrics(metrics []metricsConfig) []string {
 	return errors
 }
 
-func validateSymbol(symbol symbolConfig, metricConfig metricsConfig) []string {
+func validateEnrichSymbol(symbol *symbolConfig, metricConfig *metricsConfig) []string {
```

Comment:
could it be `validateAndEnrichSymbol` ?

Reply:
Wanted to keep the name short enough and consistent with existing `validateEnrichMetricTag`, `validateEnrichMetrics`, `validateEnrichMetricTags`.